index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORTFOLIO</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header class="header_container">
+        <nav class="navbar">
+            <h2>Welcome</h2>
+            <img src="/assets/Union.png" alt="bg">
+        </nav>
+        <section class="intro_section">
+            <h2> Hey there. I'm Favour <br>
```

Comment:
Consider making this `h2` tag an `h1` tag as it would make more sense. `h1` tags are reserved only for the most important heading on the whole page, and I am sure you would agree with me that introducing yourself is the most important part.